core/src/main/res/values/strings.xml

```diff
@@ -478,7 +478,7 @@
     <string name="found_in_shownotes_label">Found in show notes</string>
     <string name="found_in_chapters_label">Found in chapters</string>
     <string name="found_in_authors_label">Found in authors</string>
-    <string name="found_in_feeds_label">Found in feeds</string>
+    <string name="found_in_feeds_label">Found in podcasts</string>
```

Comment:
Shouldn't this be singular just like "Found in title"?

Reply:
I guess the reasoning for all the 'found in...' strings is that multiple chapters, show notes etc are searched. So I guess this should be plural as well, albeit we could make it more explicit with 'podcast identifiers' (although I doubt its usefullness).

Reply:
Isn't this text shown below every search result? So it tells that this search result is presented because the string was found in the title or the shownotes. Every item only has one feed, so I think this should be singular

Reply:
Hmm, fair enough. But then shouldn't all these strings be singular?

Reply:
Each episode contains multiple chapters and I think the word "shownotes" is usually used in plural, too. So they seem to be correct

Reply:
Thanks, done.